third_party/move/move-compiler-v2/tests/bytecode-generator/pack_order.exp

```diff
@@ -1,4 +1,4 @@
-// ---- Model Dump
+// -- Model dump before bytecode pipeline
```

Comment:
I'm not sure it was useful to change this message, as it's clearly the final Model.